packages/client/components/AvatarList.tsx

```diff
@@ -42,21 +46,30 @@ interface Props {
   emptyEl?: ReactElement
   isAnimated?: boolean
   borderColor?: string
+  // the current container may not be sized so that we can observe some parent container
+  containerRef?: React.RefObject<HTMLDivElement>
```

Comment:
We need to somehow be able to watch some parent container size, because the HTML layout may not guarantee that the current container will be an adequate size. 
I can't find a better solution for this. 🤷🏻‍♂️